yarn.lock

```diff
@@ -820,9 +820,9 @@ heroku-client@3.0.6, heroku-client@^3.0.6:
     is-retry-allowed "^1.0.0"
     tunnel-agent "^0.6.0"
 
-heroku-exec-util@0.7.3:
+"heroku-exec-util@https://github.com/heroku/heroku-exec-util#start-checking-host-keys":
```

Comment:
i guess lock files don't work well with arbitrary dependency versions :/

Reply:
It does the job! 😄 🤷‍♂